src/policy_evaluator/policy_evaluator_builder.rs

```diff
@@ -206,7 +206,8 @@ impl PolicyEvaluatorBuilder {
                 || {
                     let mut wasmtime_config = wasmtime::Config::new();
                     if self.wasmtime_cache {
-                        wasmtime_config.cache_config_load_default()?;
+                        let cache = wasmtime::Cache::new(wasmtime::CacheConfig::new()).unwrap();
```

Comment:
Do not `unwrap`, it leads to panics.

```suggestion
                        let cache = wasmtime::Cache::new(wasmtime::CacheConfig::new())?;
```

Reply:
Amended. Felt safe, given that we just consume the `CacheConfig::new()`, but I see the maintainability problem.